inception-doc/src/main/resources/META-INF/asciidoc/user-guide.adoc

```diff
@@ -100,6 +100,14 @@ include::{include-dir}monitoring.adoc[leveloffset=+1]
 
 <<<
 
+include::{include-dir}workload_settings.adoc[leveloffset=+1]/home/thefarseer/Desktop/workload.png
```

Comment:
the inclusion of the workload_settings has not worked. I would also consider merging the settings section and the workload page into one section, because you need to understand one to enable the other. I fear users might only read the section on the workload page and then wonder why it is not showing up.